tests/system/presenter/test_get_forwarding_meetings.py

```diff
@@ -0,0 +1,107 @@
+        self.assertEqual(status_code, 200)
+        self.assertEqual(
+            data,
+            [
+                {
+                    "id": 3,
+                    "name": "com3",
+                    "meetings": [{"id": 2, "name": "meeting2"}],
+                }
+            ],
+        )
+
+    def test_no_permissions(self) -> None:
+        self.set_models(
+            {
+                "user/3": {
+                    "username": "test",
+                    "is_active": True,
+                    "default_password": TEST_USER_PW,
+                    "password": self.auth.hash(TEST_USER_PW),
+                    "group_$3_ids": [3],
+                },
+                "meeting/3": {"group_ids": [3]},
+                "group/3": {"meeting_id": 3},
+            }
+        )
+        self.client.login("test", TEST_USER_PW)
+        status_code, data = self.request("get_forwarding_meetings", {"meeting_id": 3})
+        assert status_code == 403
+        assert "Missing permission" in data["message"]
+
+    def test_complex(self) -> None:
+        self.set_models(
+            {
+                "meeting/1": {"name": "meeting1", "committee_id": 2, "group_ids": [2]},
+                "meeting/2": {"name": "meeting2", "committee_id": 3, "group_ids": [3]},
+                "meeting/3": {"committee_id": 3, "group_ids": [4]},
+                "committee/2": {
+                    "name": "com2",
+                    "forward_to_committee_ids": [3],
+                    "meeting_ids": [1],
+                },
+                "committee/3": {
+                    "name": "com3",
+                    "meeting_ids": [2, 3],
+                },
+                "user/3": {
+                    "username": "test",
+                    "is_active": True,
+                    "default_password": TEST_USER_PW,
+                    "password": self.auth.hash(TEST_USER_PW),
+                    "group_$1_ids": [2],
+                    "group_$2_ids": [3],
+                    "group_$3_ids": [4],
+                },
+                "group/2": {
+                    "meeting_id": 1,
+                    "permissions": [Permissions.Motion.CAN_MANAGE],
+                },
+                "group/3": {
+                    "meeting_id": 2,
+                    "permissions": [Permissions.Motion.CAN_CREATE],
+                },
+                "group/4": {
+                    "meeting_id": 3,
+                    "permissions": [],
+                },
+            }
+        )
+        self.client.login("test", TEST_USER_PW)
+        status_code, data = self.request("get_forwarding_meetings", {"meeting_id": 1})
+        self.assertEqual(status_code, 200)
+        self.assertEqual(
+            data,
+            [
+                {
+                    "id": 3,
+                    "name": "com3",
+                    "meetings": [{"id": 2, "name": "meeting2"}],
+                }
+            ],
+        )
```

Comment:
Please add a new test (or add it to this test) where a meeting is excluded from the result because the user does not have the CAN_CREATE permission in any of its meetings.

Reply:
This is checked with meeting/3 asaik. It is excluded, because of the missing permission.

Reply:
No, meeting 3 belongs to committee 3 which is included in the result. What I meant is adding a fourth committee with it's own meeting in which the user does not have the `motion.can_create` permission and then asserting that the additional committee is not included in the result.

Reply:
Updated the complex. Add another meeting / committee without perms. The additional committee is not in the results.